app/controllers/reviews_controller.rb

```diff
@@ -18,6 +20,44 @@ def create
     end
   end
 
+  def upvote
+    @review = Review.find(params[:id])
+    @review.upvote_by current_user
+
+    respond_to do |format|
+      if @review.upvote_by current_user
+        format.json {
+            render json: {
+            id: @review.id,
+            upsize: @review.get_upvotes.size,
+            downsize: @review.get_downvotes.size
+          }
+        }
+      else
+        render json: {}
+      end
+    end
+  end
+
+  def downvote
+    @review = Review.find(params[:id])
+    @review.downvote_by current_user
+
+    respond_to do |format|
+      if @review.downvote_from current_user
+        format.json {
+            render json: {
+            id: @review.id,
+            downsize: @review.get_downvotes.size,
+            upsize: @review.get_upvotes.size
+          }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.